mne/stats/tests/test_parametric.py

```diff
@@ -15,68 +15,84 @@
     # R 15.2
     # data generated using this code http://goo.gl/7UcKb
     'r_fvals': np.array([2.567619, 0.24006, 1.756380]),
-    'r_pvals_uncorrected': np.array([0.12557, 0.78776, 0.1864])
+    'r_pvals_uncorrected': np.array([0.12557, 0.78776, 0.1864]),
+    # and https://gist.github.com/dengemann/5539403
+    'r_fvals_3way': np.array([
+        0.74783999999999995,   # A
+        0.20895,               # B
+        0.21378,               # A:B
+        0.99404000000000003,   # C
+        0.094039999999999999,  # A:C
+        0.11685,               # B:C
+        2.78749]),              # A:B:C
+    'r_fvals_1way': np.array([0.67571999999999999])
 }
```

Comment:
@haribharadwaj see SPSS and R tests, 
for the latest R tests: https://gist.github.com/dengemann/5539403